srcpkgs/subtitlecomposer/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'subtitlecomposer'
+pkgname=subtitlecomposer
+version=0.7.1
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules kcoreaddons kdoctools gettext pkg-config
+ pkg-config qt5-host-tools qt5-qmake kconfig xorg-server-xvfb"
+makedepends="blas-devel ffmpeg-devel jack-devel ktextwidgets-devel kio-devel
+ ktextwidgets-devel kcodecs-devel kross-devel kxmlgui-devel ki18n-devel
+ libopenal-devel pocketsphinx sonnet-devel"
+depends="kcoreaddons ktextwidgets kio kcodecs kross kxmlgui ki18n libopenal
+ ffmpeg sonnet"
```

Comment:
> [pocketsphinx v5](https://github.com/cmusphinx/pocketsphinx) is currently not released (except for a release candidate which isn't tagged)

maybe take a release from sourceforge. idk.
https://sourceforge.net/projects/cmusphinx/files/pocketsphinx/5prealpha/

Reply:
based on what's said in the github repo, that's not a stable release, I think.

Reply:
> You shouldn't need to specify the `depends` because they'll get auto-detected, and you had some unnecessary things in `hostmakedepends`

I still add `kconfig` in `hostmakedepends` so it can still compile.

https://github.com/void-linux/void-packages/runs/7597149424?check_suite_focus=true#step:7:168

Reply:
> based on what's said in the github repo, that's not a stable release, I think.

I guess so too. hmmm..